micronota/tests/test_workflow.py

```diff
@@ -21,7 +21,7 @@ class TestAnnotate(TestCase):
     def setUp(self):
         test_dir = join('micronota', 'db', 'tests', 'data', 'uniref')
         cwd = dirname(__file__)
-        test_dir = '/Users/zech/Dropbox/git/micronota/micronota/db/tests/data/uniref'
+        test_dir = '../db/tests/data/uniref'
```

Comment:
@mortontj, like I expected, the rel path does not work:

======================================================================
ERROR: test_annotate (micronota.tests.test_workflow.TestAnnotate)
----------------------------------------------------------------------
Traceback (most recent call last):
  File
"/Users/zech/Dropbox/git/micronota/micronota/tests/test_workflow.py", line
38, in setUp
    for seq in read(files[1], format='fasta'):
  File "/Users/zech/Dropbox/git/scikit-bio/skbio/io/registry.py", line
1147, in read
    **kwargs)
  File "/Users/zech/Dropbox/git/scikit-bio/skbio/io/registry.py", line 509,
in read
    return (x for x in itertools.chain([next(gen)], gen))
  File "/Users/zech/Dropbox/git/scikit-bio/skbio/io/registry.py", line 526,
in _read_gen
    with _resolve_file(file, **io_kwargs) as (file, _, _):
  File
"/Users/zech/anaconda/envs/micronota/lib/python3.5/site-packages/contextlib2.py",
line 49, in __enter__
    return next(self.gen)
  File "/Users/zech/Dropbox/git/scikit-bio/skbio/io/util.py", line 204, in
_resolve_file
    file, source, is_binary_file = _resolve(file, **kwargs)
  File "/Users/zech/Dropbox/git/scikit-bio/skbio/io/util.py", line 59, in
_resolve
    newfile = source.get_reader()
  File "/Users/zech/Dropbox/git/scikit-bio/skbio/io/_iosources.py", line
88, in get_reader
    return io.open(self.file, mode='rb')
FileNotFoundError: [Errno 2] No such file or directory:
'../db/tests/data/uniref/uniref100_Swiss-Prot_Bacteria.fna'


On Tue, Mar 1, 2016 at 12:19 PM, Jamie Morton <notifications@github.com>
wrote:

> In micronota/tests/test_workflow.py
> <https://github.com/RNAer/micronota/pull/2#discussion_r54628816>:
>
> > @@ -21,7 +21,7 @@ class TestAnnotate(TestCase):
> >      def setUp(self):
> >          test_dir = join('micronota', 'db', 'tests', 'data', 'uniref')
> >          cwd = dirname(__file__)
> > -        test_dir = '/Users/zech/Dropbox/git/micronota/micronota/db/tests/data/uniref'
> > +        test_dir = '../db/tests/data/uniref'
>
> This will work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RNAer/micronota/pull/2/files#r54628816>.
>